vault.py

```diff
@@ -485,14 +485,19 @@ def api_vault_system_metadata(ctx, coll):
         landinpage_url = row[0]
         system_metadata["Landingpage"] = "<a href=\"{}\">{}</a>".format(landinpage_url, landinpage_url)
 
+    # Check for base DOI
```

Comment:
Lets discuss with the team, but maybe we can drop the DOIs from the system metadata if we always display the versions? Also when only a single version DOI exists?

Reply:
Removed DOIs from system metadata section.